src/components/DocumentVideo/PaperIdFlowSelector/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { h, FunctionComponent } from 'preact'
+import { memo } from 'preact/compat'
+
+import { useLocales } from '~locales'
+import style from './style.scss'
+
+import type { CaptureFlows } from '~types/docVideo'
+import type { DocumentTypes } from '~types/steps'
+
+type IdFlows = Extract<CaptureFlows, 'cardId' | 'paperId'>
+
+type IdFlowButtonProps = {
+  idType: IdFlows
+  onClick: () => void
+  title: string
+}
+
+const TITLE_KEY_BY_DOCUMENT_TYPE: Partial<Record<DocumentTypes, string>> = {
+  driving_licence: 'doc_capture.prompt.title_license',
+  national_identity_card: 'doc_capture.prompt.title_id',
+}
+
+const BUTTON_COPY_BY_ID_TYPE = {
+  cardId: 'doc_capture.prompt.button_card',
+  paperId: 'doc_capture.prompt.button_paper',
+}
+
+const IdFlowButton: FunctionComponent<IdFlowButtonProps> = ({
+  idType,
+  onClick,
+  title,
+}) => (
+  <button className={style[idType]} onClick={onClick}>
+    <span className={style.icon} />
+    <span className={style.text}>{title}</span>
+    <span className={style.chevron} />
+  </button>
+)
```

Comment:
I might be wrong and maybe it doesn't quite suit in with the use case here in the end, but the [Castor Button component can also be used with icon](https://onfido.github.io/castor/?path=/docs/react-button--with-icon).

Reply:
Humm great, but how about both icon & a chevron?

Reply:
You should be fine to do this
```
<Button kind="action" variant="secondary">
    <span className={style.icon} />
    <span className={style.text}>{title}</span>
    <span className={style.chevron} />
</Button>
```